lib/chef/provider/aws_rds_instance.rb

```diff
@@ -36,14 +36,12 @@ def destroy_aws_object(instance)
     end
     # Wait up to 10 minutes for the db instance to shutdown
     converge_by "waited until RDS instance #{new_resource.name} was deleted" do
-      wait_for(
-        aws_object: instance,
-        query_method: :exists?,
-        expected_responses: [false],
-        acceptable_errors: [AWS::RDS::Errors::DBInstanceNotFound],
-        tries: 60,
-        sleep: 10
-      )
+      tries, sleep = 60, 10
```

Comment:
The exists? method no longer exists, and I couldn't find another way to detect whether the database had been deleted. The expected_response to the ```instance.load``` is the ```::Aws::RDS::Errors::DBInstanceNotFound``` exception but I didn't see a way to make ```wait_for``` use it the right way. If you have a better way, I'm all ears.